tests/sitemaps_tests/test_http.py

```diff
@@ -185,18 +184,6 @@ def test_sitemap_get_urls_no_site_2(self):
         with self.assertRaises(ImproperlyConfigured):
             Sitemap().get_urls()
 
-    def test_sitemap_item(self):
```

Comment:
The thing is that this is testing `Sitemap.get_urls()` -- nothing `GenericSitemap` specific -- that's why I was wondering if the test could be rewritten not to use `GenericSitemap`.